xml/System.Windows.Forms/SystemInformation.xml

```diff
@@ -4184,10 +4114,8 @@
 |<xref:System.Windows.Forms.SystemInformation.IsMenuAnimationEnabled%2A>|Indicates whether menu fade or slide animation features are enabled.|  
 |<xref:System.Windows.Forms.SystemInformation.MenuAccessKeysUnderlined%2A>|Indicates whether menu access keys are always underlined.|  
 |<xref:System.Windows.Forms.SystemInformation.IsSelectionFadeEnabled%2A>|Indicates whether the selection fade effect is enabled.|  
-|<xref:System.Windows.Forms.SystemInformation.IsToolTipAnimationEnabled%2A>|Indicates whether ToolTip animation is enabled.|  
-  
-> [!NOTE]
->  This property is supported only on Windows 2000, Windows XP, and the Windows Server 2003 family. On other platforms, this property returns `false`.  
+|<xref:System.Windows.Forms.SystemInformation.IsToolTipAnimationEnabled%2A>|Indicates whether ToolTip animation is enabled.|
+ On other platforms, this property returns `false`.  
```

Comment:
@RussKie since the property was supported only in operating systems that are currently unsupported, should the fixed sentence be "This property always returns `false`"?

Reply:
I think now the doc is wrong, as mentioned previous all these comments omitted "_...and later_" part.
I've tested this property locally on .NET 5.0, and it returns `true`:
![image](https://user-images.githubusercontent.com/4403806/112258177-2192b300-8cba-11eb-85cf-a0bfab1ccee3.png)